keyboards/keebio/levinson/keymaps/issmirnov/rgb.c

```diff
@@ -0,0 +1,66 @@
+#include "rgb.h"
+
+// TODO gate this debugging header
+#include <print.h>
+
+// Wired up in layer_state_set_user in keymap.c
+uint32_t layer_state_set_rgb(uint32_t state) {
```

Comment:
We've changed this function to use typedef variables, so that they can be changed (eg, support fewer layers)
```suggestion
layer_state_t layer_state_set_rgb(layer_state_t state) {
```

Reply:
Done.